src/data/action.ts

```diff
@@ -30,6 +30,10 @@ export const ACTION_ICONS = {
   wait_template: mdiCodeBraces,
   wait_for_trigger: mdiTrafficLight,
   repeat: mdiRefresh,
```

Comment:
Should we remove the normal repeat? Is it used still?

Reply:
Yeah It's used to render the ui mode from YAML_ONLY_ACTION_TYPES

Without this the repeat action becomes yaml only